ui/apps/platform/src/Containers/NetworkGraph/utils/flowUtils.ts

```diff
@@ -33,6 +34,21 @@ export function getNumFlows(flows: Flow[]): number {
     return numFlows;
 }
 
+export function getUniqueIdFromFlow(flow: Flow) {
+    const { entityId, direction, port, protocol } = flow;
+    const id = `${entityId}-${direction}-${port}-${protocol}`;
+    return id;
+}
+
+export function getUniqueIdFromPeer(peer: Peer) {
+    const { id: entityId } = peer.entity;
+    const direction = peer.ingress ? 'Ingress' : 'Egress';
+    const { port } = peer;
+    const { protocol } = peer;
+    const id = `${entityId}-${direction}-${port}-${protocol}`;
```

Comment:
nit: Extract this concatenation into a reusable function

(can be done in later PR)